cmd/girafSort/girafSort_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/giraf"
+	"testing"
+)
+
+func TestGirafSort(t *testing.T) {
+	graphFile := "../../sort/testdata/mini.gg"
+	readsFile := "../../sort/testdata/miniReads.giraf"
+	outFile := "miniReads_sorted_test.giraf"
```

Comment:
Would probably be good to keep files be read or written during testing to the testdata directory

Reply:
I opted for this since there is no testdata directory, but I see your point. Would you feel ok with ioutil.TempFile?